src/lib.rs

```diff
@@ -42,8 +42,8 @@
 //!
 //! ## Rust Version
 //!
-//! This version of itertools requires Rust 1.32 or later.
-#![doc(html_root_url="https://docs.rs/itertools/0.8/")]
+//! This version of itertools requires Rust 1.36 or later.
+#![doc(html_root_url="https://docs.rs/itertools/0.11/")]
```

Comment:
I wonder why we have this line at all. I've never written this in any of my other crates. Perhaps it's just a vestige of the past that can be removed without issue?